### PR TITLE
Update Flink / Kubernetes demo for 1.12

### PR DESCRIPTION
**Pull request by yankcrime**
Most of the updates to this demo are with regards to the deployment of MKE and how that's subsequently configured to expose the Kubernetes API - in this case I've used Marathon-LB.

It's been tested with DC/OS 1.12.2 and MKE 2.2.0-1.13.3.

### CODE REVIEW

File: flink-k8s/1.12/README.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+Transaction{timestamp=1520472906000, origin=2, target='3', amount=8613}
+Transaction{timestamp=1520473004000, origin=2, target='3', amount=5027}
+Transaction{timestamp=1520473050000, origin=2, target='3', amount=924}
+Transaction{timestamp=1520473177000, origin=2, target='3', amount=1566}
+Transaction{timestamp=1520473412000, origin=2, target='3', amount=5272}}
+```
+
+### Helm
+
+You can also install `Generator` and `Viewer` with the Helm.
```
Comment (mattj-io) [inline flink-k8s/1.12/README.md]: Guessing this wasn't your typo but 'the Helm' should just be Helm

File: flink-k8s/1.12/webui/webui_viewer.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
+
```
Comment (mattj-io) [inline flink-k8s/1.12/webui/webui_viewer.go]: Never seen these files before, looks like a UI - but no reference to this in the docs ..

File: flink-k8s/1.12/README.md (hunk 1 above)
Comment (mattj-io) [inline flink-k8s/1.12/README.md]: See above

File: flink-k8s/1.12/webui/webui_viewer.go (hunk 2 above)
Comment (yankcrime) [reply flink-k8s/1.12/webui/webui_viewer.go]: As discussed I've removed it from this demo and will clean up the rest in a separate PR.  It'll be -re-added at a later date.

**Pull request merged**